scripts/digimap-ci.coffee

```diff
@@ -6,10 +6,14 @@
 #   hubot: dmci deploy <app> <version> <site> - Deploy an  app (defaults: version=latest, site=beta).
 #   hubot: dmci release <app> - Tell Jenkins what to run the Maven Release Plugin for <app>.
 #   hubot: dmci update <days> - Update beta with apps deployed in last <days> (default 1 day).
+#   hubot: dmci start update <days> <cron> - Regularly deploy to beta looking over the past <days> with period specified in cron format by <cron>
```

Comment:
This line is too long, do a help when running locally, but otherwise, the deploy command is as long as it can go with wrapping. What about: *Deploy to beta apps deployed <days> ago using cron syntax*